.circleci/config.yml

```diff
@@ -401,7 +406,9 @@ commands:
 
 jobs:
   verify-code:
-    executor: ubuntu
+    executor:
+      name: ubuntu
+      machine: xlarge
```

Comment:
curious what machine is used by default if `machine` is not specified

Reply:
[medium](https://github.com/mapbox/mapbox-maps-android/blob/main/.circleci/config.yml#L683)